src/index.js

```diff
@@ -104,17 +104,17 @@ export default class SaucelabsConnector {
         await closeTunnel();
     }
 
-    async waitForFreeMachines (machinesCount, requestInterval, maxAttemptsCount) {
+    async waitForFreeMachines (machineCount, requestInterval, maxAttemptCount) {
         var attempts = 0;
 
-        while (attempts < maxAttemptsCount) {
-            var freeMachinesCount = await this._getFreeMachinesCount();
+        while (attempts < maxAttemptCount) {
+            var freeMachineCount = await this._getFreeMachineCount();

```

Comment:
This don't works. The function still defined as `_getFreeMachinesCount`... 
be attentive